### DISCUSSION TITLE
Scene is too large error on publishing when it was not too large before

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2830] [General Questions & Discussion] [answered]
I'm getting an error when I try to publish my scene. It says Scene too large, even though it was fine before and not a particularly big scene.  I tried restarting Chrome (sometimes works for some reason), but it did not resolve the error. Has anyone seen this issue and know why it happens?

**Comment by emclaren** [comment, marked as answer]
This is very strange Hilary, would you be ok to share your scene file with us at hubs@mozilla.com (you can export the .spoke file from the dropdown menu)?

**Reply by hilarytsai** [reply]
Hi @emclaren  Found the issue! It was a piece of geometry way out in the distance that I hadn't realized was there. Issue solved. Thank you!
